.github/workflows/ci.yml

```diff
@@ -16,19 +16,34 @@ jobs:
       - name: Noop
         run: "true"
   ci:
-    name: (${{ matrix.os }}) CI
+    name: (${{ matrix.name }}) CI
     needs: org-check
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
         # N.B.: macos-13 is the oldest non-deprecated Intel Mac runner and macos-14 is the oldest
         # non-deprecated ARM Mac runner.
-        os: [ ubuntu-22.04, linux-arm64, macos-13, macos-14, windows-2022, windows-arm64 ]
+        include:
+          - os: ubuntu-24.04
+            name: Linux x86-64
+          - os: linux-arm64
```

Comment:
I kept this, so we are using the linux arm runner. Let me know if you want me to use the regular ubuntu runner and then use Docker for arm64 builds as well.